# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [1]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows: 
1. The first a random number from 1 to 4 
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [2]:
series1 = pd.Series(np.random.randint(1,5,100))

In [3]:
series2 = pd.Series(np.random.randint(1,4,100))

In [4]:
series3 = pd.Series(np.random.randint(10000,30001,100))

### Step 3. Let's create a DataFrame by joinning the Series by column

In [6]:
DataFrame = pd.concat([series1,series2,series3],axis=1)
DataFrame

0  1      2
0   2  1  19668
1   3  1  10363
2   2  2  17495
3   4  2  26145
4   4  2  11614
.. .. ..    ...
95  2  3  12594
96  3  1  22795
97  4  1  11049
98  1  2  18880
99  3  3  26151

[100 rows x 3 columns]

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [7]:
DataFrame.columns = ['bedrs','bathrs','price_sqr_meter']
DataFrame

bedrs  bathrs  price_sqr_meter
0       2       1            19668
1       3       1            10363
2       2       2            17495
3       4       2            26145
4       4       2            11614
..    ...     ...              ...
95      2       3            12594
96      3       1            22795
97      4       1            11049
98      1       2            18880
99      3       3            26151

[100 rows x 3 columns]

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [8]:
bigcolumn = pd.DataFrame(pd.concat([series1,series2,series3],ignore_index=True,axis=0))
bigcolumn

0
0        2
1        3
2        2
3        4
4        4
..     ...
295  12594
296  22795
297  11049
298  18880
299  26151

[300 rows x 1 columns]

### Step 6. Oops, it seems it is going only until index 99. Is it true?

### Step 7. Reindex the DataFrame so it goes from 0 to 299

In [9]:
bigcolumn.reset_index()

index      0
0        0      2
1        1      3
2        2      2
3        3      4
4        4      4
..     ...    ...
295    295  12594
296    296  22795
297    297  11049
298    298  18880
299    299  26151

[300 rows x 2 columns]